### DISCUSSION TITLE
Error when trying to use RSVP on event page

### DISCUSSION THREAD

**Discussion by WXQ8NPb5TrbDNqzywcxF** [liedekef/events-made-easy#646] [General]
Hi,

I'm tryinkg to use booking form on the event page.
I don't know how to debug it,  but when i press the button to confirm the booking,  it return yellow square with an error, and the booking is not saved.

I've tried to enable wp_debug without success.
I've tried to disable most of the plugin of wordpress without success.
Email sent well with "test email" option

**Comment by WXQ8NPb5TrbDNqzywcxF** [comment]
Sorry, seem to be cloudflare problem.

**Reply by liedekef** [reply]
ok, so not an issue then? If it is an issue, please open a new issue (not a discussion).
